static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -53,13 +75,40 @@ export const getEpicTestToRun = memoize(
                     test => !test.highPriority
                 );
 
-                return firstRunnableTest<EpicABTest>([
+                const result = firstRunnableTest<EpicABTest>([
                     hardCodedPriorityEpicTest,
                     ...highPriorityConfiguredTests,
                     ...highPriorityHardCodedTests,
                     ...lowPriorityConfiguredTests,
                     ...lowPriorityHardCodedTests,
                 ]);
+
+                if (config.get('switches.compareVariantDecision')) {
```

Comment:
Sorry for the maybe obvious question: The switch name here is camelCase but it's dash-case when the switch is defined. Do they need to match up? (Good idea to add a switch btw :+1:)

Reply:
Yeah, for some reason it converts it to this somewhere before it gets into the config object on the client side.